packages/connect/src/events/device.ts

```diff
@@ -13,12 +14,21 @@ export const DEVICE = {
     CHANGED: 'device-changed',
     FIRMWARE_VERSION_CHANGED: 'device-firmware_version_changed',
 
+    // This event is triggered every time, the device provides the THP credentials to the Suite.
+    // This happens on two occasions:
+    //      1) User just entered the Code and the device provided the pairing credentials.
+    //      2) User initiated the autoconnect flow and confirmed on the device the autoconnect.
+    //         Device then responded with a new credential that allows the Suite to autoconnect.
+    THP_CREDENTIALS_CHANGED: 'device-thp_credentials_changed',
+
     // trezor-link events in protobuf format
     BUTTON: 'button',
     PIN: 'pin',
     PASSPHRASE: 'passphrase',
     PASSPHRASE_ON_DEVICE: 'passphrase_on_device',
     WORD: 'word',
+    THP_AUTOCONNECT: 'thp_autoconnect', // ask UI if autoconnect credentials should be created
```

Comment:
remove this, THP_AUTOCONNECT will not be used, it was from the variant when autoconnect credentials were generated immediately after pairing